### PR TITLE
pre-commit script add

### PR DESCRIPTION
**Pull request by maddhruv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,8 +56,14 @@
...
   },
   "devDependencies": {
+    "husky": "^1.2.1",
     "jest": "^23.6.0",
     "markdown-magic": "^0.1.25",
     "standard": "^12.0.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run test && npm run markdown"
```
Comment (hemal7735) [inline package.json]: running `npm run markdown` is not needed, as we already validate in `npm run test`.
```suggestion
      "pre-commit": "npm run test"
```

Comment (hemal7735) [reply package.json]: Is there a way for telling husky to not include `unstage` files while running hooks?

Comment (bnb) [reply package.json]: If y'all can come to a conclusion on this, this can be merged 👍

**Comment by bnb** [comment]
Going to close this since changes have been left unaddressed. Feel free to address and re-open. That said, at this point I'm not _entirely_ sure what problem this is trying to solve.

**Pull request closed without merging**